### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by TuringRs**

### CODE REVIEW

File: src/main/java/mops/foren/domainmodel/Post.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package mops.foren.domainmodel;
+
+import java.sql.Date;
+import java.util.List;
+
+public class Post {
+    User user;
+    Date date;
+    String text;
```
Comment (niruc100) [inline src/main/java/mops/foren/domainmodel/Post.java]: please remove String text

File: src/main/java/mops/foren/domainmodel/User.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package mops.foren.domainmodel;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class User {
+    String name;
+    String email;
+    Set<String> roles = new HashSet<>();
+    String image;
```
Comment (niruc100) [inline src/main/java/mops/foren/domainmodel/User.java]: change Attributes to Microtypes

File: src/main/java/mops/foren/domainmodel/User.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package mops.foren.domainmodel;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class User {
+    String name;
+    String email;
+    Set<String> roles = new HashSet<>();
```
Comment (TuringRs) [inline src/main/java/mops/foren/domainmodel/User.java]: Roles shouldn't be Strings

**Review by TuringRs — COMMENTED** [review]
For the first example it's ok, but we should keep in mind that there are many attribute types to change.

**Review by niruc100 — APPROVED** [review]
is good enough

**Pull request merged**